docs/Client_Server.md

```diff
@@ -0,0 +1,39 @@
+# Client and Server
+
+## Precompile
+
+Server side we precompile all our React components into static HTML.
+
+Use the `npm install` command to trigger or update the precompile.
+
+### Webpack
+
+Webpack creates static HTML pages from a list of react routes.
```

Comment:
It does sound like I am saying this is a feature of webpack.

Something like "In our webpack.config we create static HTML pages from a list of react routes."

I'm not sure I want to mention the plugin yet, In that line, I'm trying to stick to what we're doing, not how.